src/drivers/env.ts

```diff
@@ -34,4 +34,5 @@ export const config = {
       env.P0_GOOGLE_OIDC_CLIENT_SECRET ?? "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",
   },
   appUrl: env.P0_APP_URL ?? "https://api.p0.app",
+  environment: env.P0_ENV ?? "production",
```

Comment:
Should this be `NODE_ENV`? (Not sure what the correct answer is here, maybe this is better).

Reply:
wanted to make sure user's NODE_ENV settings do not impact p0 cli settings in production.